README.md

```diff
@@ -29,6 +38,19 @@ git clone https://github.com/sap/service-fabrik-boshrelease
 cd service-fabrik-boshrelease
 ```
 
+### Deploying ApiServer
+Apiserver is a prerequisite for running Service-Fabrik-Broker. It can be deployed on boshlite using provided manifest [file](https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/master/templates/apiserver.yml)
```

Comment:
Apiserver uses ETCD as backend. So, ETCD is also a prerequisite.

`Apiserver along with ETCD` can be deployed...............

Reply:
I needed to update bosh-cloudconfig with below address before deploying apiserver. Is this step somewhere else?
```
static:
  - 10.244.0.2 - 10.244.14.254

bosh -e boshlite update-cloud-config cloud-config.yml
```

Reply:
cloud-config present in boshrelease is outdated. Raised PR #116  with working cc